#! /usr/bin/env python

PACKAGE='aero_ssd_recognition'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("window_size", int_t, 0, "valid point search area size for simple mode", 10, 3, 30)
gen.add("search_step", int_t, 0, "valid point search step for simple mode", 2, 1, 50)
gen.add("mode", int_t, 0, "3D point estimation mode. 0: mean 1: sandwich 2: nearest 3: nearest caenter", 0, 0, 3)

exit(gen.generate(PACKAGE, "aero_ssd_recognition", "Object3DProjector"))
